### PR TITLE
ENH: Added default logger, Timm class uses kwargs

### PR DESCRIPTION
**Pull request by emherk**
Includes #190 #199 
- added default logger implementation
- refactored Timm class to use kwargs
- moved spacing printer to the logger's file, it was only used in logger anyways

### CODE REVIEW

**Comment by emherk** [comment]
WIP

File: Procfile
Comment (schobbejak) [inline Procfile:1]: What is this for?

**Comment by schobbejak** [comment]
Looks fine, need to fix the conflicts and then I can approve. Next time try to keep the issues separate in different PRs

File: Procfile
Comment (emherk) [reply Procfile:1]: I was editing the logger, and I see no reason why it should have an underscore

Comment (schobbejak) [reply Procfile:1]: The Procfile

**Pull request merged**